hasher-matcher-actioner/webapp/src/pages/SubmitContent.jsx

```diff
@@ -100,134 +101,128 @@ export default function SubmitContent() {
   };
 
   return (
-    // ToDo header copied from matches page -> standardize into shared component
-    <div className="d-flex flex-column justify-content-start align-items-stretch h-100 w-100">
-      <div className="flex-grow-0">
-        <Container className="bg-dark text-light" fluid>
-          <Row className="d-flex flex-row justify-content-between align-items-end">
-            <div className="px-4 py-2">
-              <h1>Submit Content</h1>
-              <p>Provide content to the HMA system to match against.</p>
-              <p>
-                Currently only <b>images</b> using the submisson type{' '}
-                <b>Direct Upload</b> or <b>URL</b> are supported.
-              </p>
-            </div>
-            <div className="px-4 py-2" />
-          </Row>
-        </Container>
-      </div>
-      <Container className="mt-4" style={{overflowY: 'auto'}}>
```

Comment:
Could you double check that adding a bunch of additional fields is still visible/scrollable? this override was originally to allow for that but it might not be needed anymore

Reply:
I'll check.

Reply:
Feel free to just create an issue if so and I can fix it in my next PR (I want to update "metadata" to be 
"additional fields" anyway.